lib/usage.dart

```diff
@@ -26,19 +26,34 @@ library usage;
 
 import 'dart:async';
 
+import 'src/usage_impl_default.dart'
+   if (dart.library.js) 'src/usage_impl_html.dart'
+   if (dart.library.ui) 'src/usage_impl_flutter.dart'
```

Comment:
It's interesting that the Flutter import analyzes correctly; `flutter` isn't listed in the pubspec. Does the analyzer only look  at the default import (`src/usage_impl_default.dart`)?